data-prepper-api/src/main/java/org/opensearch/dataprepper/model/event/Event.java

```diff
@@ -143,11 +143,12 @@ public interface Event extends Serializable {
      * of a Data Prepper expression
      * @param format input format
      * @param expressionEvaluator - The expression evaluator that will support formatting from Data Prepper expressions
+     * @param replacementForFailures - The String to use as a replacement for when keys in Events can't be found
```

Comment:
Should we just call this `defaultValue`?

Reply:
Yeah that does make more sense I think